### ISSUE TITLE
Fails when request hits a Premium Quad Stash Tab

### ISSUE THREAD

**Issue by atsay714** [TehGM/PoE-QualityRecipesCalculator#2]
```
Failed downloading stash data
System.ArgumentException: An item with the same key has already been added. Key: Requires {1} (Level {0})
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at TehGM.PoE.QualityRecipesCalculator.Serialization.ItemPropertiesConverter.ReadJson(JsonReader reader, Type objectType, Object existingValue, JsonSerializer serializer)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.DeserializeConvertable(JsonConverter converter, JsonReader reader, Type objectType, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(IList list, JsonReader reader, JsonArrayContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Populate(JsonReader reader, Object target)
   at Newtonsoft.Json.JsonSerializer.PopulateInternal(JsonReader reader, Object target)
   at Newtonsoft.Json.JsonSerializer.Populate(JsonReader reader, Object target)
   at TehGM.PoE.QualityRecipesCalculator.Serialization.SerializationHelper.PopulateObject[T](JToken token, T target, String childPath, JsonSerializer serializer)
   at TehGM.PoE.QualityRecipesCalculator.PoeHttpClient.GetStashTabsAsync(String league, CancellationToken cancellationToken)
   at TehGM.PoE.QualityRecipesCalculator.Program.<>c.<<Main>b__0_0>d.MoveNext()```

**Comment by TehGM** [comment]
Thank you for the report. Will investigate and try to fix.

Note to self: caused by an item having the same property twice - didn't think it's possible.

**Comment by atsay714** [comment]
I lied, went back and double checked now that you mentioned it. It seems to be caused by Heist blueprints (not contracts), rather than the quad stash tab. It's just all my blueprints were in my quad tab.

**Comment by TehGM** [comment]
I added a fix for this in b392a59 and Release 1.0.2, but I do not have any data to test with (Dyson Sphere Program successfuly pulled me away from PoE recently). Whenever you have time, do let me know if you still notice the issue!

**Comment by TehGM** [comment]
Closing as there has been no response.
